string pipe_category

string pipe_name

bool use_only_local_segments

# Normally left empty but when its required that the pipe must publish
# certain data to specific topics, then these are described in here. 
diagnostic_msgs/KeyValue[] output_topics

# Specifies certain required parameters of a pipe
temoto_component_manager/PipeSegmentSpecifier[] pipe_segment_specifiers

# Normally left empty, used by context manager interface status callback
string pipe_id

# Resource Registrar metadata
temoto_resource_registrar/TemotoMetadata TemotoMetadata

---

# Topics where raw tracking data is published. By default the data
# is published on one topic but there could be exceptions.
diagnostic_msgs/KeyValue[] output_topics

# The id of the pipe, e.g., /pipe_123_at_robot_3/
string pipe_id

# Resource Registrar metadata
temoto_resource_registrar/TemotoMetadata TemotoMetadata
